src/library/driftory.ts

```diff
@@ -23,12 +23,12 @@ const osdPromise = new Promise((resolve, reject) => {
 });
 
 type Frame = any;
```

Comment:
@iangilman do you think you can look into typing the `Frame` type correctly?

I thought it might be the frame object from the json but it's more complicated than that.

I just typed it as `any` because i didn't want to have to try and figure it out. It should really be given proper types information though.

Reply:
I think it's literally just `OpenSeadragon.Rect`, but I'll take a look.